downstream/modules/platform/ref-controller-api-config-settings.adoc

```diff
@@ -0,0 +1,78 @@
+[id="controller-api-config-settings"]
+
+= Configuration settings
+
+There are two named-URL-related configuration settings available under `/api/v2/settings/named-url/: NAMED_URL_FORMATS` and `NAMED_URL_GRAPH_NODES`.
+
+`NAMED_URL_FORMATS` is a read only key-value pair list of all available named URL identifier formats. 
+The following shows an example `NAMED_URL_FORMATS`:
+
+[literal, options="nowrap" subs="+attributes"]
+----
+"NAMED_URL_FORMATS": {
+"organizations": "<name>",
+"teams": "<name>++<organization.name>",
```

Comment:
Is 'name' specific to the parameter type? Otherwise everything with '<name>++<organization.name>' will be identical.